src/components/Modal/Modal.styles.ts

```diff
@@ -62,7 +62,7 @@ export const StyledModalFeet = styled.div`
 export const StyledMobileModal = styled.div`
   z-index: 999;
   width: 100%;
-  height: fit-content;
```

Comment:
Question; have we done full re-test of iOS, android etc after changing this? I think there were a number of issues with the previous value and this was only changed 4 weeks ago. @njomb perhaps your input here since you changed this recently?

Reply:
Testing was done on some pages that use modal using developer tools, was the previous value height 100%?

Reply:
Previous value before `fit-content` was `100vh`. This had a number of bugs across alert modal and other modals in some platforms. Have we done a full regression test of all modals on major platforms?

Reply:
`fit-content` should be the value here. Developer tools won't show the true view because of the way the web app is rendered in the iOS app. both 100vh and 100% will cause the modal to overflow out of bounds and hide the submit button